prime-dotnet-webapi/Models/Agreements/AgreementType.cs

```diff
@@ -19,5 +23,12 @@ public static bool IsEnrolleeAgreement(this AgreementType agreementType)
         {
             return agreementType != AgreementType.CommunityPracticeOrgAgreement && agreementType != AgreementType.CommunityPharmacyOrgAgreement;
         }
+
+        public static IEnumerable<AgreementType> EnrolleeAgreementTypes()
+        {
+            return Enum.GetValues(typeof(AgreementType))
+                .Cast<AgreementType>()
+                .Where(t => t.IsEnrolleeAgreement());
```

Comment:
This isn't really an extension method, this is a static helper method. From a technical perspective, there isn't much difference but the methods are called using a different syntax. Extension methods have additional syntactic sugar that makes them look like instance methods on objects rather than static methods. `IsEnrolleeAgreement()`, for instance, can be called like 

```C#
var agreementType = AgreementType.OboToa;
agreementType.IsEnrolleeAgreement();
```
rather than

```C#
var agreementType = AgreementType.OboToa;
AgreementTypeExtensions.IsEnrolleeAgreement(agreementType);
```

This is just as an aside, this code is fine for now

Reply:
Since enum could not hold any method so borrowed some estate from the extension class for the helper method😀